samples/edge_impulse/data_forwarder/README.rst

```diff
@@ -17,7 +17,7 @@ The sample supports the following development kits:
 
 .. table-from-rows:: /includes/sample_board_rows.txt
    :header: heading
-   :rows: nrf9160dk_nrf9160_ns, nrf52dk_nrf52832, nrf52840dk_nrf52840
+   :rows: nrf9160dk_nrf9160_ns, nrf5340dk_nrf5340_cpuapp_and_cpuapp_ns, nrf52dk_nrf52832, nrf52840dk_nrf52840
```

Comment:
nrf5340dk_nrf5340_cpuapp_and_cpuapp_ns ? what on earth is this thing?

Reply:
Its just a simplification for generating the table if secure and non-secure is supported.

Reply:
> Its just a simplification for generating the table if secure and non-secure is supported.

It would be simpler to just mention boards... One would not have to wonder if this is not some strange board variant. The sole fact that it was mentioned in review tells how odd it is.

Anyway. Approved :)